l10n_eu_product_adr/tests/test_adr_goods.py

```diff
@@ -160,57 +158,3 @@ def test_05_product_variant_create(self):
         self.assertTrue(
             var.adr_goods_id == self.goods1 for var in template.product_variant_ids
         )
-
-    # def test_06_product_variant_migration(self):
-    #     """Test the migration of the adr fields from template to product"""
-    #     try:
-    #         from openupgradelib import openupgrade  # noqa: F401
-    #     except ImportError:
-    #         logging.getLogger("odoo.addons.l10n_eu_product_adr.tests").info(
-    #             "OpenUpgrade not found, skip test"
```

Comment:
I wouldn't drop this test, as it probably had meaning in the past.
It is probably meant to be activated again once code is fixed.